snap_plugin/v1/tests/mock_plugins.py

```diff
@@ -156,3 +156,74 @@ def stop(self):
         self._stopper.set()
         self.stopped = True
         self.stop_plugin()
+
+
+class MockStreamCollector(snap.StreamCollector, threading.Thread):
+    """Mock streaming plugin """
+
+    def __init__(self, name, ver):
+        super(MockStreamCollector, self).__init__(name, ver)
+        self._flags.add('require-config', snap.plugin.FlagType.toggle, '')
```

Comment:
This flag was added to MockCollector for testing diagnostics, it's not needed here.